<?xml version="1.0" encoding="UTF-8"?><testrun name="ch.ethz.fgremper.cloudstudio.testing.combination" project="CSTesting" tests="8" started="8" failures="1" errors="0" ignored="0">
  <testsuite name="ch.ethz.fgremper.cloudstudio.testing.combination.BranchLevelAwarenessTest" time="5.367">
    <testcase name="testRelationWithOrigin" classname="ch.ethz.fgremper.cloudstudio.testing.combination.BranchLevelAwarenessTest" time="4.04"/>
    <testcase name="testBranch" classname="ch.ethz.fgremper.cloudstudio.testing.combination.BranchLevelAwarenessTest" time="1.326">
      <failure>
        <expected>NOT_CHECKED_OUT</expected>
        <actual>REMOTE_BRANCH</actual>org.junit.ComparisonFailure: expected:&lt;[NOT_CHECKED_OUT]&gt; but was:&lt;[REMOTE_BRANCH]&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at ch.ethz.fgremper.cloudstudio.testing.combination.BranchLevelAwarenessTest.testBranch(BranchLevelAwarenessTest.java:162)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
  </testsuite>
  <testsuite name="ch.ethz.fgremper.cloudstudio.testing.combination.FileLevelAwarenessTest" time="7.942">
    <testcase name="testBranchCompare" classname="ch.ethz.fgremper.cloudstudio.testing.combination.FileLevelAwarenessTest" time="4.843"/>
    <testcase name="testBranchInternal" classname="ch.ethz.fgremper.cloudstudio.testing.combination.FileLevelAwarenessTest" time="3.099"/>
  </testsuite>
  <testsuite name="ch.ethz.fgremper.cloudstudio.testing.combination.ContentLevelConflictTest" time="7.533">
    <testcase name="testCorrectConflictDetection" classname="ch.ethz.fgremper.cloudstudio.testing.combination.ContentLevelConflictTest" time="2.293"/>
    <testcase name="testCorrectFileContentAndConflictCount" classname="ch.ethz.fgremper.cloudstudio.testing.combination.ContentLevelConflictTest" time="3.649"/>
    <testcase name="testNoCommonAncestor" classname="ch.ethz.fgremper.cloudstudio.testing.combination.ContentLevelConflictTest" time="1.59"/>
  </testsuite>
  <testsuite name="ch.ethz.fgremper.cloudstudio.testing.combination.TestThreeUsersRunClient" time="0.318">
    <testcase name="test" classname="ch.ethz.fgremper.cloudstudio.testing.combination.TestThreeUsersRunClient" time="0.318"/>
  </testsuite>
</testrun>
